### ISSUE TITLE
[stripe][android] Selecting a payment method results in crash

### ISSUE THREAD

**Issue by msn444** [triniwiz/nativescript-plugins#162]
With 9.0.0-beta.1, standard integration -- selecting a payment method results in the following crash:

```
System.err: An uncaught Exception occurred on "main" thread.
  System.err: Calling js method onPaymentSessionDataChanged failed
  System.err: Error: Missing getter method for property: type
  System.err:
  System.err: StackTrace:
  System.err: createPaymentMethod(file: app/webpack:/app/node_modules/@triniwiz/nativescript-stripe/standard/index.android.js:285:0)
  System.err:   at onCustomerRetrieved(file: app/webpack:/app/node_modules/@triniwiz/nativescript-stripe/standard/index.android.js:257:0)
  System.err:   at onPaymentSessionDataChanged(file: app/webpack:/app/node_modules/@triniwiz/nativescript-stripe/standard/index.android.js:254:0)
```

Looking at `sessionData.paymentMethod` just before `createPaymentMethod` is called, it reports missing getters for all properties:
![image](https://user-images.githubusercontent.com/24908050/211696147-83044407-31ef-4d4c-98b8-83209f9e8e54.png)

**Comment by triniwiz** [comment]
Which version runtime are you using ?

**Comment by msn444** [comment]
Hi @triniwiz, I was running @nativescript/android 8.3.1. I've now upgraded to 8.4.0 and the issue is solved. Thanks!

**Comment by triniwiz** [comment]
Yes that kotlin getter issue was fixed in 8.4 😅
